llama_stack/providers/utils/telemetry/tracing.py

```diff
@@ -6,7 +6,7 @@
 
 import asyncio
 import contextvars
-import logging
+import logging  # allow-direct-logging
```

Comment:
why not using `get_logger` here, too ? It's imported some lines below.

Reply:
I have no strong opinion here, would be fine with either way. You can keep this in mind and send in a followup PR on this, I think that would be cleaner and gets this PR over the finish line quicker.

Reply:
lgtm 👍🏽